buildscripts/buildvars-setup.cmd

```diff
@@ -139,17 +139,29 @@ call "%_VSCOMNTOOLS%\VsDevCmd.bat"
 if "%VisualStudioVersion%"=="15.0" (
     goto :VS2017
 )
+if "%VisualStudioVersion%"=="16.0" (
+    goto :VS2019
+)
 
 :MissingVersion
-:: Can't find VS 2017
-echo Visual Studio 2017 is a pre-requisite to build this repository.
+:: Can't find VS 2017 ^/ 2019
+echo Visual Studio 2017 or 1029 is a pre-requisite to build this repository.
```

Comment:
Typo: 1029

Reply:
Hehe, thanks for spotting, will fix :-).